creator.html

```diff
@@ -10,21 +10,16 @@
                 <div class="mask rgba-black-light d-flex justify-content-center align-items-center">
                     <div class="text-center white-text mx-5 wow fadeIn">
                         <h1 class="mb-4">
-                            <strong>Convert your ideas into apps with Kodular!</strong>
+                            <strong>Convert Your Ideas Into Apps With Kodular!</strong>
```

Comment:
```suggestion
                            <strong>Convert your ideas into apps with Kodular Creator!</strong>
```

1. This can NOT be capitalized, it looks bad as there are too many words with capitals